loader/src/injector/hook.cpp

```diff
@@ -1176,6 +1205,7 @@ static void unhook_functions() {
         }
     }
     delete plt_hook_list;
+    plt_hook_list = NULL;
```

Comment:
Will this make any difference aside from making it more clearer later on? A comment would be nice